### ISSUE TITLE
read only -> read write -> not syncing

### ISSUE THREAD

**Issue by shoeper** [haiwen/seafile-client#655]
It seems like the client does not recognize a permission change.

When I share a library read only, sync it and decide later on that read-write permission would be better. The client does not recognize the permission change.

**Comment by mathieu-mp** [comment]
It did the job in my case, but previously read-only client made lots of duplicates as soon as it became read-write... Had to restore at state just before changing rights.

**Comment by killing** [comment]
This is currently a limitation. When there're many libraries on the server,
checking a library's permission on the server regularly would be heavy load
to the database. So we decided not to check it. You can resync the library.
That should do the trick.

2015-07-08 23:00 GMT+08:00 Sven <notifications@github.com>:

**Comment by shoeper** [comment]
I don't accept it as valid answer because the library were not shown as read only in seafile clients library list. So the client did still knew it, but didn't change the behavior.

**Comment by killing** [comment]
Yeah, you do inspire me a solution to this problem. Since GUI refresh the
list every few minutes, we can take this chance to update the local
permission.

2015-07-09 16:20 GMT+08:00 Sven <notifications@github.com>:

**Comment by killing** [comment]
This was fixed with later version updates.
